Intersect.Server/Entities/Entity.cs

```diff
@@ -1571,21 +1571,14 @@ byte projectileDir
             }
 
             //Handle DoT/HoT spells]
+            
             if (spellBase.Combat.HoTDoT)
             {
-                var doTFound = false;
-                foreach (var dot in target.CachedDots)
-                {
-                    if (dot.SpellBase.Id == spellBase.Id && dot.Target == this)
-                    {
-                        doTFound = true;
-                    }
-                }
+                target.CachedDots.ToList()
```

Comment:
Can you provide performance metrics for the benefit of using linq here?
Generally speaking it's a much slower alternative to a regular foreach loop with an if.

Reply:
Nope lol.

In general, I prefer LINQ because of its readability, writeability, and the fact that by leveraging an external lib, you leave the door open for improvements to be made down the line without you doing anything.

I found a good article, however, on performance here: https://medium.com/swlh/is-using-linq-in-c-bad-for-performance-318a1e71a732

This all being said, LMAO this was a dumb implementation because I was doing two loops.